app/javascript/packages/session/index.ts

```diff
@@ -0,0 +1,32 @@
+import { request } from '@18f/identity-request';
+
+export interface SessionStatusResponse {
+  /**
+   * Whether the session is still active.
+   */
+  live: boolean;
+
+  /**
+   * ISO8601-formatted date string for session timeout.
+   */
+  timeout: string;
+}
+
+export const STATUS_API_ENDPOINT = '/active';
+export const KEEP_ALIVE_API_ENDPOINT = '/sessions/keepalive';
+
+/**
+ * Request the current session status. Returns a promise resolving to the current session status.
+ *
+ * @return A promise resolving to the current session status
+ */
+export const requestSessionStatus = () => request<SessionStatusResponse>(STATUS_API_ENDPOINT);
+
+/**
+ * Request that the current session be kept alive. Returns a promise resolving to the updated
+ * session status.
+ *
+ * @return A promise resolving to the updated session status.
+ */
+export const requestSessionKeepAlive = () =>
```

Comment:
the name is tripping me up a bit here, "request that something be kept alive", but also this is small and happy to merge this as-is

I feel like we're marking the session as still active?

- `markSessionActive`
- `extendCurrentSession`

Reply:
In my head, I was trying to think of a conventional prefix we might use for an asynchronous behavior which involves a network request, and also aligning the naming to the underlying `request` mechanic. That being said, maybe it's not appropriate to expose the implementation detail of there being a network request, so the package API could opt for something more readable.

Reply:
maybe we could namespace it in whatever we import from?

```
import { extendCurrentSession } from 'session/network-requests'
```

Reply:
Renamed to `extendSession` in 4a6e75aca. I think I land more on the side here that a consumer wouldn't need to care whether the operation happens as the result of a network request. I kept the name `requestSessionStatus` since I think that's a reasonably clear function name regardless of it happening through a network request, though open to suggestions if a better name fits.

Reply:
I would consider `sessionStatus` 

But yes you're right, since it's a promise, as long as the client knows to handle the async part of that, they don't need to worry network vs not

Reply:
Hm, yeah, that's nice and concise, but lacks a verb prefix I might conventionally expect from (most) function names in JS.